docs/develop/support-ie-11.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Support Internet Explorer 11
+description: 'Learn how to support Internet Explorer 11 and ES5 Javascript in your add-in.'
+ms.date: 05/19/2021
+localization_priority: Normal
+---
+
+# Support Internet Explorer 11
+
+Office Add-ins are web applications that are displayed inside IFrames when running on Office on the web. Office Add-ins are displayed using embedded browser controls when running in Office on Windows or Office on the Mac. The embedded browser controls are supplied by the operating system or by a browser installed on the user's computer.
+
+If you plan to market your add-in through AppSource or you plan to support certain older versions of Windows and Office, your add-in must work in the embeddable browser control that is based on Internet Explorer 11 (called "Trident"). For information about which combinations of Windows and Office use the Internet Explorer 11 browser control, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
+
+> [!IMPORTANT]
+> Internet Explorer 11 doesn't support some HTML5 features such as media, recording, and location. If your add-in must support Internet Explorer 11 (see the second paragraph), then you can't use these features.
+
+Internet Explorer 11 doesn't support JavaScript versions later than ES5. If you want to use the syntax and features of ECMAScript 2015 or later, or TypeScript, you have two options as described in this article. (You can also combine these two techniques.)
```

Comment:
Would it be worthwhile to combine this paragraph and "Important" note? They both are describing stuff that's not supported in IE11. Maybe there could be a bulleted list of unsupported features, like:
- JavaScript versions ES6 and later
- TypeScript
- Media
- Recording
- Location